source/agora/node/Registry.d

```diff
@@ -1031,7 +1028,7 @@ private struct ZoneData
     {
         if (this.type == ZoneType.primary)
         {
-            this.soa.serial = cast(uint) Clock.currTime(UTC()).toUnixTime();
+            this.soa.serial += 1;
```

Comment:
Using the UNIX time is quite handy though, to know about the last update.
Perhaps `max(time, serial + 1)` would be the best of both worlds ?